addons/website_slides/static/src/xml/website_slides_channel.xml

```diff
@@ -8,38 +8,44 @@
-                            Training (with progress)
-                      </span>
+                <label for="channel_type">Choose a layout</label>
+                <div class="form-row">
+                    <div class="form-group col-6">
+                        <div class="form-check px-0">
+                            <input class="form-check-input d-none" type="radio" name="channel_type" id="channel_type1" value="training" checked="checked"/>
+                            <label for="channel_type1">
+                                <img class="w-100" src="/website_slides/static/src/img/channel-training-layout.png" alt="Training Layout"/>
+                            </label>
+                        </div>
                     </div>
-                    <div class="form-check">
-                      <input class="form-check-input" type="radio" name="channel_type" id="channel_type2" value="documentation"/>
-                      <span class="form-check-label" for="channel_type2">
-                            Documentation
-                      </span>
+                    <div class="form-group col-6">
+                        <div class="form-check px-0">
+                            <input class="form-check-input d-none" type="radio" name="channel_type" id="channel_type2" value="documentation"/>
+                            <label for="channel_type2">
+                                <img class="w-100" src="/website_slides/static/src/img/channel-documentation-layout.png" alt="Documentation Layout"/>
+                            </label>
+                        </div>
                     </div>
-                </fieldset>
+                </div>
                 <div class="form-group">
                     <label for="title">Description</label>
-                    <input type="text" class="form-control" name="description" id="description" placeholder="Write here a short description of your first course" />
+                    <textarea rows="2" class="form-control" name="description" id="description"
+                              placeholder="Common tasks for a computer scientist is asking the right questions and answering
+                              questions. In this course, you'll study those topics with activities about mathematics, science and logic." />
                 </div>
                 <div class="form-group">
                     <label id="communication-label">Review</label>
                     <div class="o_wslide_channel_communication_type">
                         <div class="form-check">
                             <input class="form-check-input" type="checkbox" id="allow_comment" name="allow_comment"/>
-                            <span class="form-check-label" for="allow_comment">Allow review on Course</span>
+                            <span class="form-check-label" for="allow_comment">Allow students to review your courses</span>
```

Comment:
course without a 's' I suppose ?

Reply:
Well seen David.
Can you mark it as checked by default as well?